spec/models/book_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+describe Book do
+  before do
+    Book.delete_all
+  end
+
+  describe ".ordered" do
+    it "orders records by name ASC" do
+      beloved = Book.create!(name: "Beloved", isbn: 1987654321)
+      animal_farm = Book.create!(name: "Animal Farm", isbn: 1234567890)
+
+      expect(described_class.ordered).to eq([animal_farm, beloved])
+    end
```

Comment:
Other cases I'd be keen to know about:

- Is a book named animal Farm going to show before or after a book named Animal Farm?
- Is a book named animal Farm going to show before or after a book named Beloved?
- Is a book named beloved going to show before or after a book named Animal Farm?

At least one of these fails in a way you might not expect.